src/tools/kdb/complete.cpp

```diff
@@ -210,35 +227,49 @@ void CompleteCommand::printResult (const Key originalRoot, const Key root, const
 		cout << endl;
 	}
 
-	if (cl.debug)
+	for (const auto & it : hierarchy)
 	{
-		cout << endl << "Dumping whole analyzation results" << endl;
-		for (const auto & it : hierarchy)
+		if (filterPredicate (it.first.getFullName ()) && it.second.second > minDepth && it.second.second <= maxDepth)
 		{
-			cout << it.first << " " << it.second.first << " " << it.second.second << endl;
+			printResult (it, cl.verbose);
 		}
-		cout << endl;
 	}
+}
 
-	for (const auto & it : hierarchy)
+void CompleteCommand::printResult (const pair<Key, pair<int, int>> & current, const bool verbose)
+{
+	cout << current.first;
+	if (current.second.first > 1)
 	{
-		if (filterPredicate (it.first.getFullName ()) && it.second.second > minDepth && it.second.second <= maxDepth)
+		cout << "/ ";
```

Comment:
right, of course this makes not much sense, moved the additional whitespace to the verbose output where it separates the number of children from the keyname.